nf-core-contributors.yaml

```diff
@@ -485,3 +485,16 @@ contributors:
       contact_github: benbfly
       location: [31.764656, 35.150221]
       twitter: Hujimed
+    - full_name: National Institute of Biological Standards and Control
+      short_name: NIBSC
+      affiliation: Analytical and Biological Sciences
+      description: >
+        The National Institute for Biological Standards and Control (NIBSC) plays a leading national and international role in assuring the quality of biological medicines and diagnostics.
+      address: Blanche Ln, South Mimms, Potters Bar EN6 3QG, United Kingdom
+      url: https://nibsc.org
+      image_fn: NIBSC_logo_rgb_screen.svg
+      contact: Francesco Lescai
+      contact_email: francesco.lescai@nibsc.org
+      contact_github: nibscles
+      location: [51.688315, -0.240508]
+      twitter: tokybo
```

Comment:
This is supposed to be the institutional twitter.
Does your institute has any?

Reply:
changed :)